zerver/lib/actions.py

```diff
@@ -1430,6 +1430,27 @@ def _internal_prep_message(realm, sender, recipient_type_name, parsed_recipients
 
     return None
 
+def _internal_prep_message_to_user_ids(realm, sender, recipient_ids, content):
+    # type: (Realm, UserProfile,  List[int], Text) -> Optional[Dict[str, Any]]
+
+    if len(content) > MAX_MESSAGE_LENGTH:
+        content = content[0:3900] + "\n\n[message was too long and has been truncated]"
+
+    if realm is None:
+        raise RuntimeError("None is not a valid realm for _internal_prep_message_to_user_ids!")
+
+    try:
+        return check_message(sender, get_client("Internal"), 'private', '',
+                             content, realm=realm, recipient_ids=recipient_ids)
+    except JsonableError as e:
+        logging.error(u"Error queueing internal message by %s: %s" % (sender.email, e))
+
+    return None
+
+def internal_prep_message_to_user_ids(realm, sender, recipient_ids, content):
+    # type: (Realm, UserProfile, List[int], Text) -> Optional[Dict[str, Any]]
+    return _internal_prep_message_to_user_ids(realm, sender, recipient_ids, content)
```

Comment:
This one-line wrapper seems unneeded.  If we are exposing a public version of this function (i.e. not using the "_" convention), then we can have everybody call the public version, including functions inside the module.